resources/views/components/input-field.blade.php

```diff
@@ -0,0 +1,29 @@
+@props([
+    "type" => "text",
+    "id",
+    "placeholder",
+    "variant" => "input",
+])
+
+<div class="flex flex-col gap-2 w-full max-w-[20.75rem] sm:max-w-[38rem]">
```

Comment:
```suggestion
<div class="flex flex-col gap-2 w-full">
```

Can add max-w on parent depending on where its needed